python-sdk/nuscenes/predict/input_representation/static_layers.py

```diff
@@ -77,6 +84,123 @@ def correct_yaw(yaw: float) -> float:
 
     return yaw
 
+def get_lanes_in_radius(x: float, y: float, radius: float,
+                        discretization_meters: float,
+                        map_api: NuScenesMap) -> Dict[str, List[Tuple[float, float, float]]]:
+    """
+    Retrieves all the lanes and lane connectors in a radius of the query point.
+    :param x: x-coordinate of point in global coordinates
+    :param y: y-coordinate of point in global coordinates
+    :param radius: Any lanes within radius meters of the (x, y) point will be returned.
+    :param discretization_meters: How finely to discretize the lane. If 1 is given, for example,
+        the lane will be discretized into a list of points such that the distances between points
+        is approximately 1 meter.
+    :param map_api: The NuScenesMap instance to query.
+    :return: Mapping from lane id to list of coordinate tuples in global coordinate system.
+    """
+
+    lanes = map_api.get_records_in_radius(x, y, radius, ['lane', 'lane_connector'])
+    lanes = lanes['lane'] + lanes['lane_connector']
+    lanes = map_api.discretize_lanes(lanes, discretization_meters)
+
+    return lanes
+
+def color_by_yaw(agent_yaw_in_radians: float,
+                 lane_yaw_in_radians: float) -> Color:
+    """
+    Color the pose one the lane based on its yaw difference to the agent yaw.
+    """
+
+    # By adding pi, lanes in the same direction as the agent are colored blue.
+    angle = angle_diff(agent_yaw_in_radians, lane_yaw_in_radians, 2*np.pi) + np.pi
+
+    # Convert to degrees per colorsys requirement
+    angle = angle * 180/np.pi
+
+    normalized_rgb_color = colorsys.hsv_to_rgb(angle/360, 1., 1.)
+
+    color = tuple([color*255 for color in normalized_rgb_color])
+    return color
```

Comment:
Tuple or Color?

Reply:
I'm now explicitly unpacking the tuple so that it is consistent with the return type